ui/src/components/DashboardInventory.test.js

```diff
@@ -0,0 +1,183 @@
+//@flow
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import DashboardInventory from './DashboardInventory';
+import {
+  waitForLoadingToFinish,
+  render,
+  FAKE_CONTROL_PLANE_IP,
+} from './__TEST__/util';
+import type { Alert } from '../services/alertUtils';
+import { useHighestSeverityAlerts } from '../containers/AlertProvider';
+
+const alertsCritical = [
+  {
+    id: 'alert1',
+    severity: 'critical',
+  },
+];
+const alertsWarning = [
+  {
+    id: 'alert2',
+    severity: 'warning',
+  },
+];
+const noAlerts = [];
+
+jest.mock('../containers/ConfigProvider', () => ({
+  __esModule: true, // this property makes it work
+  default: ({ children }) => <>{children}</>,
+  useConfig: () => ({
+    url: 'mock.k8s.url',
+  }),
+}));
+
+jest.mock('../containers/AuthProvider', () => ({
+  __esModule: true, // this property makes it work
+  default: ({ children }) => <>{children}</>,
+  useAuth: () => ({
+    id_token: 'mock.token',
+  }),
+}));
+
+jest.mock('../containers/PlatformLibraryProvider', () => ({
+  __esModule: true, // this property makes it work
+  default: ({ children }) => <>{children}</>,
+  usePlatformLibrary: () => ({
+    getVolumesCountQuery: () => ({
+      queryKey: 'countVolumes',
+      queryFn: () => 4,
+    }),
+    getNodesCountQuery: () => ({
+      queryKey: 'countNodes',
+      queryFn: () => 6,
+    }),
+  }),
+}));
+
+jest.mock('../containers/AlertProvider', () => ({
+  __esModule: true, // this property makes it work
+  default: ({ children }) => <>{children}</>,
+  useHighestSeverityAlerts: jest.fn(),
+  useAlertLibrary: () => ({
+    getNodesAlertSelectors: () => {},
+    getVolumesAlertSelectors: () => {},
+  }),
+  highestAlertToStatus: (alerts?: Alert[]): string => {
```

Comment:
I wonder if it would work like this:
`highestAlertToStatus: () => 'healthy',`

Reply:
@ChengYanJin Well this one does not always return "healthy" in the tests. We do expect "warning" or "critical" in the corresponding tests.